Chip-8 Challenge/src/CHIP-8 Virtual Machine/VM.cs

```diff
@@ -46,7 +48,8 @@ public VM(IKeypadMap keypadMap = null)
             _display = new Display(this);
             _delayTimer = new Timer();
             _soundTimer = new Timer();
-
+            _clockTick = new System.Timers.Timer();
+            _clockTick.Stop();
```

Comment:
This is probably not needed. I was seeing some odd behaviour on Blazor preview where the timer seemed to be running on creation.